README.md

```diff
@@ -0,0 +1,17 @@
+# otm-website-tests
+
+An automated test suite for the OpenTreeMap website.
+
+## Usage
+
+To execute the test suite, first install [Leiningen](http://leiningen.org/), then execute the following command:
+
+```bash
+$ lein midje
```

Comment:
Addressed in https://github.com/OpenTreeMap/otm-website-tests/commit/e632a44a6a31a8500ba7eab3658c0cbea424be37.